### PR TITLE
mitchell/Add nimc

### PR DESCRIPTION
**Pull request by mitchell-deriv**

### CODE REVIEW

File: src/javascript/app/pages/user/account/authenticate.js [hunk 1]
```diff
@@ -1019,9 +1087,10 @@ const Authenticate = (() => {
 
         const is_fully_authenticated = identity.status === 'verified' && document.status === 'verified';
         const should_allow_resubmission = needs_verification.includes('identity') || needs_verification.includes('document');
-        onfido_unsupported = !identity.services.onfido.is_country_supported;
+        country_code = identity.services.onfido.country_code;
+        onfido_unsupported = !identity.services.onfido.is_country_supported || country_code === 'NGA';
         const documents_supported = identity.services.onfido.documents_supported;
-        const country_code = identity.services.onfido.country_code;
+
```
Comment (mustofa-binary) [inline src/javascript/app/pages/user/account/authenticate.js:1093]: nit space

File: src/templates/app/_includes/authenticate_message.jsx [hunk 2]
```diff
@@ -98,6 +128,22 @@ const FileSelector = ({
...
                         </div>
+                        { type === 'nigeria' && (
+                            <React.Fragment>
+                                <p className='font-s'>{it.L('Upload your selfie')}</p>
+                                <div className='files'>
+                                    <h3>{it.L('Your selfie')}</h3>
+                                    <div className='fields'>
+                                        <div className='gr-row form-row gr-centered'>
+                                            <div className='gr-12'>
+                                                <input id='selfie' className='file-picker' type='file' accept='.jpg, .jpeg, .gif, .png' data-type='other' data-name={it.L('Your selfie')} data-page-type='photo' />
```
Comment (mustofa-binary) [inline src/templates/app/_includes/authenticate_message.jsx:139]: are we accepting gif 😅

Comment (mitchell-deriv) [reply src/templates/app/_includes/authenticate_message.jsx:139]: dont know but up in the same there is this and guess who did it?:)

    const accepted_formats_regex = /selfie/.test(file.passthrough.class)
            ? /^(PNG|JPG|JPEG|GIF)$/i
            : /^(PNG|JPG|JPEG|GIF|PDF)$/i;

Comment (mitchell-deriv) [reply src/templates/app/_includes/authenticate_message.jsx:139]: mm don't know should I remove? I saw that in authenticate.js there is this thats seems accepting gif also on file uploads and you are the creator seems. What should I do? thanks amami:)

      const accepted_formats_regex = /selfie/.test(file.passthrough.class)
            ? /^(PNG|JPG|JPEG|GIF)$/i
            : /^(PNG|JPG|JPEG|GIF|PDF)$/i;

**Review by mustofa-binary — DISMISSED** [review]

**Pull request closed without merging**